packages/components/src/Banner/Banner.stories.tsx

```diff
@@ -7,6 +7,14 @@ import type { Story } from "@storybook/react";
 export default {
   title: "Banner",
   component: Banner,
+  argTypes: {
+    elevation: {
+      control: {
+        type: "radio",
+        options: [0, 1],
```

Comment:
Storybook is having trouble when an argument has value `0` 🤔  this fixes the explicit control, but the elevation arg is still not passed properly when the story is embedded: https://chanzuckerberg.github.io/edu-design-system/iframe.html?id=banner--elevation-0&args=&viewMode=story

Reply:
Interesting. I noticed an issue with enums, such as

```ts
enum Cats {
  Anakin;
  Prosecco;
}
```

Maybe this is the same issue you found, since the default values will be `0` and `1`.